src/engine/Server.cpp

```diff
@@ -539,6 +539,47 @@ auto Server::setupCancellationHandle(
       std::move(cancellationHandle), std::move(cancelCancellationHandle)};
 }
 
+// _____________________________________________________________________________
+Awaitable<void> Server::sendStreamableResponse(
+    const ad_utility::httpUtils::HttpRequest auto& request, auto& send,
+    ad_utility::MediaType mediaType, const PlannedQuery& plannedQuery,
+    const QueryExecutionTree& qet,
+    SharedCancellationHandle cancellationHandle) const {
+  auto responseGenerator = ExportQueryExecutionTrees::computeResultAsStream(
+      plannedQuery.parsedQuery_, qet, mediaType, cancellationHandle);
```

Comment:
Doesn't sonar tell you to either `move` the handle here or to make it a `const &` (the moving is more idiomatic, but less likely to be suggested by the tool:)